composer.json

```diff
@@ -32,7 +32,10 @@
         "symfony/symfony": "2.6.*",
         "willdurand/hateoas-bundle": "0.3.*",
         "symfony/swiftmailer-bundle": "~2.3",
-        "pagerfanta/pagerfanta": "~1.0"
+        "pagerfanta/pagerfanta": "~1.0",
+        "sulu-io/document-manager": "dev-master",
+        "sulu/document-manager": "dev-master",
+        "dantleech/decorator-generator": "dev-master"
```

Comment:
I didn't really get the description of this library... Does it just alias class names? It doesn't provide an opportunity to further extend the new class?